src/frontend/src/lib/utils/auth.utils.ts

```diff
@@ -39,3 +39,11 @@ export const isAlternativeOrigin = (): boolean => {
 };
 
 export const hasDerivationOrigin = (): boolean => notEmptyString(AUTH_DERIVATION_ORIGIN);
+
+export const getDerivationOrigin = (): { derivationOrigin: string } | Record<string, never> => {
```

Comment:
Should we reflect the fact that it might return no derivation origin in the function's name?

`getOptionalDerivationOrigin` or `optionalDerivationOrigin` or better name?

Reply:
Sounds good to me, done!